matplotlibrc.template

```diff
@@ -35,6 +35,9 @@
 # You can also deploy your own backend outside of matplotlib by
 # referring to the module name (which must be in the PYTHONPATH) as
 # 'module://my_backend'.
+#
+# If you omit this parameter, it will always default to "Agg", which is a
```

Comment:
Here is what I observed on my particular installation:

- No user config file: `Qt5Agg` (the same backend listed in `data/matplotlibrc`.
- With user config file but `backend` is not set in that file: `Agg`.  Naively I would have expected `Qt5Agg` too but it seems that is not the case. This is what prompted this PR.